tests/plotting/test_plotting.py

```diff
@@ -3937,11 +3949,12 @@ def setup_plot():
     pl.iren._control_key_release()
     pl.close()
 
-    # shift right click dollys, image 9
+    # shift right click environment rotate, image 9
```

Comment:
We have a bunch of duplicate images in the image cache.  I'm thinking that it would be useful to add an option to `pytest_pyvista` to specify which image name to compare against for each image comparison.  This way, we could store only the unique images for this test.

Reply:
Tracked in https://github.com/pyvista/pytest-pyvista/issues/109